cypress/utils/dashboards/query_enhancement/constants.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const clusterName = 'test_cluster';
+export const clusterConnection = 'http://localhost:9200';
```

Comment:
lets not port this over imo. this should pull from a config and if that config is suppose to point to an external url then only the cypress config would be able to update so easily this constant is a constant and wont be reusable in cases